subsystem/kubeapi_test.go

```diff
@@ -1527,6 +1529,13 @@ var _ = Describe("[kube-api]cluster installation", func() {
 
 		}, "30s", "10s").Should(BeTrue())
 
+		By("Verify Agent labels")
+		labels[v1beta1.InfraEnvNameLabel] = infraNsName.Name
+		Eventually(func() map[string]string {
+			agent := getAgentCRD(ctx, kubeClient, key)
+			return agent.ObjectMeta.Labels
+		}, "30s", "10s").Should(Equal(labels))
```

Comment:
I think that you can decrease the interval, it will make the test shorter, 1 sec for example.

Reply:
Done